test/xpu_api/iterators/predef.iterators/move.iterators/move.iter.ops/move.iter.op.incr/post_incr.pass.cpp

```diff
@@ -23,8 +23,14 @@
 #include "support/utils.h"
 #include "support/test_macros.h"
 
+#include <type_traits>
```

Comment:
I think this include is not necessary - `oneapi/dpl/type_traits` should be enough

Reply:
Fixed